.github/workflows/api.yml

```diff
@@ -0,0 +1,52 @@
+on:
+  workflow_call:
+
+jobs:
+  test_api:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./serve-api
+    env:
+      TEST_RESULTS: /tmp/test_results
+    # Service containers to run with `container-job`
+    services:
+      # Label used to access the service container
+      postgres:
+        # Docker Hub image
+        image: postgres
+        # Provide the password for postgres
```

Comment:
Add a comment that these aren't real secrets, just for local test. Stops security getting jumpy.

Reply:
I'm pretty sure they'll still email you but we may as well try 😂

Reply:
Yeah all of this file was already in the Serve build pipeline, so I imagine they've seen it already